SwiftLSPClient/Types/Features/Reference.swift

```diff
@@ -21,7 +21,8 @@ public struct ReferenceParams: Codable {
     public let position: Position
     public let context: ReferenceContext
 
-    public init(textDocument: TextDocumentIdentifier, position: Position, context: ReferenceContext) {
+    public init(textDocument: TextDocumentIdentifier, position: Position,
+                context: ReferenceContext = ReferenceContext(includeDeclaration: false)) {
```

Comment:
What do you think about just adding a ` = false` to the next init method? Much shorter, and semantically equivalent as long as `ReferenceContext` doesn't change.

Reply:
👍 Done.